zerver/webhooks/beeminder/doc.md

```diff
@@ -0,0 +1,29 @@
+Get Beeminder notifications in zulip whenever you're going to derail from your goal!
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
```

Comment:
You can't use this macro if you are going to specify a different URL right after it! :)